packages/datagrid/src/components/default-cell-renderer/avro-renderer.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import DefaultTypeRenderer from './avro-renderer.component';
+
+function getComponent() {}
+
+describe('#DefaultTypeRenderer', () => {
+	it('should render DefaultTypeRenderer and load Injected Component stringCellRenderer', () => {
+		const wrapper = shallow(
```

Comment:
Please define the getComponent to return a dummy component and mount to really test the avro renderer.